rats-tls/samples/enclave-tls-server/server.c

```diff
@@ -16,9 +16,17 @@
 #include <netinet/in.h>
 #include <enclave-tls/api.h>
 #include <enclave-tls/log.h>
+#include <enclave-tls/utils.h>
 
-#define DEFAULT_PORT 1234
-#define DEFAULT_IP   "127.0.0.1"
+#define DEFAULT_PORT	    1234
+#define DEFAULT_IP	    "127.0.0.1"
+#define ENCLAVE_CONFIG_TOML "/opt/enclave-tls/config.toml"
+#define TOML_ENCLAVE_INFO   "enclave-info"
+#define TOML_MRSIGNER	    "mrsigner"
+#define TOML_MRENCLAVE	    "mrenclave"
```

Comment:
Please fix format issue

Reply:
In fact, it has been formated by clang-format. However I will change it